compiler/src/yul/mod.rs

```diff
@@ -16,7 +22,11 @@ pub fn compile(src: &str) -> Result<String, CompileError> {
         .values()
         .next()
     {
-        return Ok(first_contract.to_string());
+        return Ok(CompilerOutput {
+            tokens: format!("{:?}", tokens),
+            ast: format!("{:?}", fe_module),
```

Comment:
The formatting could be nicer but I think I'm just going to create an issue for that for now.